docs/learn-parachains.md

```diff
@@ -53,6 +53,30 @@ could not include that in their implementation and Polkadot would still enforce
 Parachains are not required to have their own token. If they do, is up to the parachain to make the
 economic case for their token, not Polkadot.
 
+## Parachain Hubs
+
+While Polkadot enables crosschain functionality amongst the parachains, it necessitates that there
+is some latency between the dispatch of a message from one parachain until the desitnation parachain
+recieves the message. In the optimistic scenario, the latency for this message should be at least 2
+blocks, one block for the message to be dispatched and one block for the receiving parachain to
+process and produce a block that acts upon the message. However, in some cases we may see that the
+latency for messages are higher if there is a lot of messages that are in queue to be processed, or
+if there exists no nodes that are running both of the parachain networks that can quickly gossip the
+message across the networks.
```

Comment:
As far as I understand it, all collators will have to be full nodes of the relay chain, so they will by default all be in contact with each other, no?

Reply:
That's a good point - they all should be gossiping relay chain messages to each other but I don't know if this includes the parachain messages or if there is a way for nodes to identify which parachain collator nodes other full nodes of the relay chain are also running.

Reply:
Did you find out?